spec/factories.rb

```diff
@@ -3,7 +3,7 @@
   factory :person do
     first_name 'John'
     last_name 'Doe'
-    email "john.doe#{Random.rand(1000)}@example.com"
+    sequence(:email) { |n| "john.doe#{n}@example.com" }
```

Comment:
What is this being changed? There is a unique validation on emails making sequences a little clunky if you were ever going to use the factories in the dev environment